api/manager.py

```diff
@@ -13,33 +13,198 @@
 @bp.route("/orders", methods=["GET"])
 def orders():
     db = get_db()
+    error = None
+    query = """
+        SELECT *
+        FROM ordered_cone
+        """
+    orders = db.execute(query).fetchall()
+    print(orders)
     # get all orders, or a specific number
     # if we want to get only a few at a time, we may need to have this be a bit more involved
     # so that we don't end up returning the same few records over and over
     # this will be used in account management as well
 
+    if orders is None:
+        error = "Not implemented"
+    elif len(orders == 0):
+        error = "Empty table"
 
-@bp.route("/product", methods=["GET"])
+    if error:
+        return json.dumps({"error": error})
+    else:
+        return json.dumps({"orders": orders})
+
+
+@bp.route("/product", methods=["GET", "PUT", "POST"])
 def accounting():
     if request.method == "GET":
         db = get_db()
+        error = None
         # get info on one specific product
+        query = """
+            SELECT display_name, stock, price_per_unit
+            FROM product
+            WHERE id = {search}
```

Comment:
incorrect sql syntax - double check how to do this & make sure you grab the product id from the request. this will look like: "/product/<product_id>" in the route if i remember off the top of my head and then you should just be able to access product_id as a variable but make sure u test that in case im off by a little